docs/conf.py

```diff
@@ -50,9 +51,25 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "sphinx_rtd_theme"
+pycytominer_hex = "#c24dd6"
+html_theme = "furo"
+html_theme_options = {
```

Comment:
Consider adding a reference to the documentation where these styling settings are described (for `furo` or otherwise).

Reply:
added in 787b20b25a3f122078aee982c13dfbc1dd5331ed